packages/type-utils/src/TypeOrValueSpecifier.ts

```diff
@@ -146,6 +146,22 @@ function typeDeclaredInFile(
   );
 }
 
+function typeDeclaredInPackage(
+  packageName: string,
+  declarationFiles: ts.SourceFile[],
+): boolean {
+  const typesPackageName =
+    '@types/' +
+    (packageName[0] === '@'
+      ? packageName.substring(1).replace('/', '__')
+      : packageName);
```

Comment:
[Refactor] ✨ 

```suggestion
  const typesPackageName = '@types/' + packageName.replace(/^@(.+)\//, '$1__');
```

Reply:
Thanks, I tweaked the regexp a bit more FYI